### PR TITLE
Migrate formats for Countryside Stewardship Grants

### PR DESCRIPTION
**Pull request by Rosa-Fox**
https://trello.com/c/UBM8Q7p1/531-csg

### CODE REVIEW

File: app/views/metadata_fields/_countryside_stewardship_grants.html.erb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  <%= f.label :land_use %>
+  <%= f.select :land_use, f.object.facet_options(:land_use), {}, { class: 'select2 form-control', multiple: true, data: {placeholder: 'Select a location'} } %>
+</div>
+<div class="form-group">
+  <%= f.label :tiers_or_standalone_items %>
+  <%= f.select :tiers_or_standalone_items, f.object.facet_options(:tiers_or_standalone_items), {}, { class: 'select2 form-control', multiple: true, data: {placeholder: 'Select a Tier or Standalone Item'} } %>
+</div>
+<div class="form-group">
+  <%= f.label :funding_amount %>
+  <%= f.select :funding_amount, f.object.facet_options(:funding_amount), { label: 'Funding (per unit per year)'}, { class: 'select2 form-control', multiple: true, data: {placeholder: 'Select a level of funding (per unit per year) '} } %>
```
Comment (benlovell) [inline app/views/metadata_fields/_countryside_stewardship_grants.html.erb]: This line could probably do with breaking over several lines. I think it's pretty hard to follow as-is.

File: app/models/countryside_stewardship_grant.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+class CountrysideStewardshipGrant < Document
+
+  FORMAT_SPECIFIC_FIELDS = [
```
Comment (benlovell) [inline app/models/countryside_stewardship_grant.rb]: You've used the `%i` notation elsewhere in this diff. We could take advantage of that here too, perhaps?

**Comment by Rosa-Fox** [comment]
Thanks @benlovell, do you think the `_countryside_stewardship_grants.html.erb` form looks ok now?
